packages/core/jsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "compilerOptions": {
+    "allowSyntheticDefaultImports": true,
+    "checkJs": true,
+    "jsx": "preserve",
+    "module": "commonjs",
+    "moduleResolution": "node",
+    "strictNullChecks": true,
+    "target": "es2017",
+  },
+  "exclude": [
+    "node_modules"
+  ]
+}
```

Comment:
iIs `jsconfig.json` the new preferred method over `.vscode/settings.json`? Is that what this file is? I don't mind moving to this, but do we need both?

Reply:
`.vscode/settings.json` and `jsconfig.json` have different roles.
The former is a local version of the IDE global settings, which currently has settings complementary to the eslint format we use. We can setup snippets and other canon-specific helpers that work in the IDE. This file however can only be read by VSCode.
The latter is a javascript compilation config file, a cousin of the typescript compilation config file tsconfig.json, but it doesn't have to do with typescript directly. This file is intended to tell the typescript compiler (which VSCode uses as source of Intellisense) that all the sibling files in this folder belong to a same project, so it lets VSCode do automatic refactors when a file is moved or renamed. It also tells it where to look for files, how to resolve the $app/$root aliases, and to complain about javascript errors with the squiggly red underline. This file can be interpreted by any IDE that uses intellisense as helper (I think Atom and a bunch more can do it)

Reply:
Wow this was an amazing explanation. Thank you! Can you add this file into the scaffold directory as well so all future projects will get it?